src/frontends/paddle/src/op/flip.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs flip(const NodeContext& node) {
+    const auto data_node = node.get_input("X");
+    const auto axes = node.get_attribute<std::vector<int32_t>>("axis");
+    const auto input_shape = data_node.get_partial_shape().get_shape();
+    const auto dims = static_cast<int32_t>(data_node.get_partial_shape().rank().get_length());
+    const auto dtype = data_node.get_element_type();
+    // for zero-dim input
+    PADDLE_OP_CHECK(node, (dims > 0), "Input dims must be greater than 0");
```

Comment:
Is this restriction asked by paddlepaddle or this op conversion? Would you please try to build a scaler "X" input model? If     we cannot create such a paddle model, we can remove this check here.

Reply:
Paddle flip do not support zero-dim input, I will remove this.